libcloudforensics/providers/gcp/internal/monitoring.py

```diff
@@ -91,6 +91,90 @@ def ActiveServices(self, timeframe: int = 30) -> Dict[str, int]:
               ret[service] = int(val)
     return ret
 
+
+  def GetNetworkData(
+      self, instance_ids: Optional[List[str]] = None,
```

Comment:
```suggestion
       self, instance_ids: List[str] = [],
```
Style nit: Removing Optional, and having a default of a zereo length list might be preferable here?

Reply:
Probably correct but will leave it to match the existing code patterns for now